### PR TITLE
probably more robust success test for relay

### PR DESCRIPTION
**Pull request by bengras**

### CODE REVIEW

File: slackrelay.py [hunk 1]
```diff
@@ -414,6 +414,7 @@ def main():
       # Pass event to the backend for each matching rule
       for r in matchingRules:
         logging.debug("Processing rule: %s" % r.name)
+        req = False
```
Comment (egoktas) [inline slackrelay.py]: I think this line is not needed.

File: slackrelay.py [hunk 2]
```diff
@@ -429,13 +430,13 @@ def main():
...
-        if not req:
-          logging.error("Error processing rule %s" % r.name)
-        else:
+        if req == True:
           if args.emoji_to_confirm is not None:
              emoji_add(part['channel'], part['ts'], args.emoji_to_confirm, sc)
           else:
              logging.debug("No emoji response configured")
+        else:
+          logging.error("Error processing rule %s" % r.name)
```
Comment (egoktas) [inline slackrelay.py]: I think it may be useful to also log the value of "req" in the error message.

**Review by egoktas — CHANGES_REQUESTED** [review]

File: slackrelay.py (hunk 1 above)
Comment (bengras) [reply slackrelay.py]: agreed, but ISTM on balance it is better to leave it in (conservative assignment making it likelier we won't get false confirmations due to overlooked logic flaws)

File: slackrelay.py (hunk 2 above)
Comment (bengras) [reply slackrelay.py]: Done, see next commit

**Review by egoktas — APPROVED** [review]

**Pull request merged**